modules/services/agentless-scanning/README.md

```diff
@@ -63,6 +63,7 @@ No modules.
 | <a name="input_trusted_identity"></a> [trusted\_identity](#input\_trusted\_identity) | The name of sysdig trusted identity | `string` | n/a | yes |
 | <a name="input_timeout"></a> [timeout](#input\_timeout) | Stackset instance timeout | `string` | `"30m"` | no |
 | <a name="mgt_stackset"></a> [mgt_stackset](#mgt\_stackset) | Whether to create the resources on the management account using a stackset | `bool` | `true` | no |
+| <a name="delegated"></a> [delegated_admin](#input\_delegated\_admin) | Whether to create the resources using an delegated admin account | `bool` | `false` | no |
```

Comment:
nit: since this is user facing input, could you please update `name="delegated"` to `name="delegated_admin"`

Reply:
Thanks I completely missed that! Fixed https://github.com/draios/terraform-aws-secure-for-cloud/pull/78/commits/771d4fe397240feabcc3045f9317520052399ffc